site/content/blog/carvelizing-helm-chart.md

```diff
@@ -0,0 +1,293 @@
+
+```bash
+$ kapp deploy -a default-ns-rbac -f https://raw.githubusercontent.com/vmware-tanzu/carvel-kapp-controller/develop/examples/rbac/default-ns.yml -y
+```
+
+**4. Install the Package**: To install a carvel Package, we need to create PackageInstall Kuberentes resource. A Package Install will install the nginx helm package and its underlying resources on a Kubernetes cluster. A `PackageInstall` references a `Package`. Thus, we can create the `PackageInstall` yaml from the `Package`.
+
+We are providing our custom values via secret. 
+
+**NOTE**: If you are using minikube, for nginx service to be in `ACTIVE` state, start `minikube tunnel` in other window as services of LoadBalancer types do not come up otherwise in minikube.  
+
+Save the below PackageInstall CR to a file named `pkginstall.yaml`
+```yaml
+---
+apiVersion: packaging.carvel.dev/v1alpha1
+kind: PackageInstall
+metadata:
+  name: nginx-pkg
+spec:
+  serviceAccountName: default-ns-sa
+  packageRef:
+    refName: nginx.bitnami.vmware.com
+    versionSelection:
+      constraints: 1.0.0
+  values:
+  - secretRef:
+      name: nginx-values
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: nginx-values
+stringData:
+  values.yml: |
+    ---
+    image:
+      pullPolicy: Always
+    serverBlock: |-
+      server {
+        listen 0.0.0.0:8080;
+        location / {
+          return 200 "Response from Custom Server";
+        }
+      }
+```
+
+```bash
+$ kapp deploy -a pkg-demo -f pkginstall.yaml -y
+```
+
+After the deploy has finished, kapp-controller will have installed the package in the cluster. We can verify this by checking the pods to see that we have a workload pod running. The output should show a single running pod which is part of nginx.
+
+```bash
+$ kubectl get pods
+```
+
+Once the pod is ready, you can use kubectl’s port forwarding to verify the customized response used in the workload.
+
+```bash
+$ kubectl port-forward service/nginx-pkg 3000:80 &
+```
+
+Now if we make a request against our service, we can see that server response is ```Response from Custom Server```
+
+```bash 
+$ curl localhost:3000
+```
+
+## Join the Carvel Community
+
+We are excited about this new adventure and we want to hear from you and learn with you. Here are several ways you can get involved:
+
+* Join Carvel's slack channel, [#carvel in Kubernetes]({{% named_link_url "slack_url" %}}) workspace, and connect with over 1000+ Carvel users.
+* Find us on [GitHub](https://github.com/vmware-tanzu/carvel). Suggest how we can improve the project, the docs, or share any other feedback.
+* Attend our Community Meetings, happening every Thursday at 10:30am PT / 1:30pm ET. Check out the [Community page](/community/) for full details on how to attend.
+
+We look forward to hearing from you and hope you join us in building a strong packaging and distribution story for applications on Kubernetes!
```

Comment:
Line 293 is copy and pasted from our KubeCon blog, and looks like it's been re-used several times on blogs since then. If this still makes sense to put on all our blogs, then keep it, otherwise change it back to simply "We look forward to hearing from you and hope you join us!"

Reply:
Removing it as we are already calling the same thing above on line: 287